front-end/src/components/Other/ShowInvite.tsx

```diff
@@ -13,17 +12,18 @@ export interface IShowInviteProps {
 }
 
 export default function ShowInvite(props: IShowInviteProps) {
-  const { showNotification } = useContext(notificationContext)
   const history = useHistory()
   const addTomRoomClickHandler = async () => {
     try {
       const { username, userId } = await addToRoom(props.roomId)
       await props.loadMessages()
       props.onJoin(username, userId)
+      console.log(props)
+      toast(`You've joined ${props.roomName}`)
```

Comment:
This prop is hardcoded in `front-end/src/Pages/Groups.tsx`. Use the actual room name instead